microbench/src/main/java/alluxio/master/file/meta/BaseInodeState.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.file.meta;
+
+import alluxio.master.file.contexts.CreateDirectoryContext;
+import alluxio.master.file.contexts.CreateFileContext;
+import alluxio.master.metastore.InodeStore;
+import alluxio.master.metastore.heap.HeapInodeStore;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class BaseInodeState {
+  protected InodeLockManager mInodeLockManager = new InodeLockManager();
+  protected InodeStore mInodeStore = new HeapInodeStore();
+
+  protected InodeDirectory mDirMnt = inodeDir(1, 0, "mnt");
+  protected InodeDirectory mRootDir = inodeDir(0, -1, "", mDirMnt);
+  protected List<Inode> mInodes = new ArrayList<>(Arrays.asList(mRootDir, mDirMnt));
+
+  protected InodeDirectory inodeDir(long id, long parentId, String name, Inode... children) {
```

Comment:
This should be split into two steps: 1 is a static utility function that creates a dir from its id and name, etc., and 2 is a member method `addInodeDir` that accepts the inode created in step 1 and its children.